sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py

```diff
@@ -17,7 +17,7 @@
 from azure.storage.blob import AccessPolicy as BlobAccessPolicy
 from azure.storage.blob import DelimitedTextDialect as BlobDelimitedTextDialect
 from azure.storage.blob import DelimitedJsonDialect as BlobDelimitedJSON
-from azure.storage.blob._generated.models import StorageErrorException
+from azure.storage.blob._generated.models import StorageErrorException  # pylint: disable=unused-import
```

Comment:
```suggestion

```

Reply:
this line could be removed, weird that pylint didn't get mad before!

Reply:
It is actually being used here https://github.com/Azure/azure-sdk-for-python/blob/00e5e5509cd55e1d57960cc77cb09329edbc4e6d/sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py#L302 I'm not sure why it's causing this pylint error so I just ignored it

Reply:
it seems on stg74 branch it's not used, while I didn't see pylint error when merging previous prs.
I just resolved conflicts on this pr and removed this line, let's see if CI is happy